copy/copy.go

```diff
@@ -95,6 +94,7 @@ type Options struct {
 	DestinationCtx   *types.SystemContext
 	ProgressInterval time.Duration                 // time to wait between reports to signal the progress channel
 	Progress         chan types.ProgressProperties // Reported to when ProgressInterval has arrived for a single artifact+offset.
+	ManifestType     string
```

Comment:
Please name this `ManifestMIMEType`, or perhaps `ForceManifestMIMEType`, and document it at least enough to say that `""` means use the default autodetection.

Reply:
fixed